app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,32 @@
+class NotificationsController < ApplicationController
+  before_action :load_noti_user, :load_notification
+
+  def update
+    if @notification.update_attributes(status: :seen)
+      respond_to :html, :js
+    else
+      respond_to do |format|
+        format.js {render "alert(#{t "notifications_controller.fail"})"}
```

Comment:
toàn alert thôi à =)), sao ko render 1 file đi

Reply:
alert là hợp lí rồi a ơi, xóa cái thông báo thôi mà làm hẳn file thì căng thẳng quá :)))